Content.Shared/Contests/ContestsSystem.cs

```diff
@@ -38,11 +41,12 @@ public float MassContest(EntityUid performerUid, bool bypassClamp = false, float
                 || performerPhysics.Mass == 0)
                 return 1f;
 
-            return _cfg.GetCVar(CCVars.AllowClampOverride) && bypassClamp
+            return Math.Clamp(_cfg.GetCVar(CCVars.AllowClampOverride) && bypassClamp
                 ? performerPhysics.Mass / otherMass
                 : Math.Clamp(performerPhysics.Mass / otherMass,
                     1 - _cfg.GetCVar(CCVars.MassContestsMaxPercentage) * rangeFactor,
-                    1 + _cfg.GetCVar(CCVars.MassContestsMaxPercentage) * rangeFactor);
+                    1 + _cfg.GetCVar(CCVars.MassContestsMaxPercentage) * rangeFactor),
+                    float.Epsilon, float.MaxValue);
```

Comment:
you should probably extract this code part in the separate method and replace all the FOURTEEN occurrences of it with this method

Reply:
Got it.